tests/test_vfs.py

```diff
@@ -83,13 +83,3 @@ def test_read_vfs_none():
             'tests/data/RGB.byte.tif') as src:
         assert src.name == 'tests/data/RGB.byte.tif'
         assert src.count == 3
-
-
-@pytest.mark.parametrize('mode', ['r+', 'w'])
-def test_update_vfs(tmpdir, mode):
-    """VFS datasets can not be created or updated"""
-    with pytest.raises(TypeError):
```

Comment:
What's the story behind removing this test? Do the new classes expose r+ w mode to vfs?